types/src/crypto.rs

```diff
@@ -3,10 +3,14 @@
 mod asymmetric_key;
 mod error;
 
+#[cfg(any(feature = "std", test))]
+pub use asymmetric_key::generate_ed25519_keypair;
 #[cfg(any(feature = "gens", test))]
```

Comment:
```suggestion
#[cfg(any(feature = "testing", feature = "gens", test))]
```

I think this change should be applied to every place in the `casper-types` crate where we feature gate behind "gens" - 8 instances I believe, including this one.

Reply:
Applied the correct feature flag and made the changes in the 7 other places